CHANGELOG.rst

```diff
@@ -0,0 +1,48 @@
+#########
+Changelog
+#########
+
+All notable changes to this project will be documented in this file.
+
+The format is based on `Keep a changelog`_ and this project adheres to
+`Semantic Versioning`_.
+
+.. _Keep a changelog: https://keepachangelog.com/en/1.0.0
+.. _Semantic Versioning: https://semver.org/spec/v2.0.0.html
+
+*************
+`Unreleased`_
+*************
+
+Added
+=====
+
+* ``IMGPROXY_SKIP``, option to skip imgproxy URL generation and only
```

Comment:
Does double backtick mean something or is this a typo?

Reply:
In rST, single backtick means link while double backtick means verbatim/code.

Ref: https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html

Reply:
👍